modules/ROOT/pages/changelog.adoc

```diff
@@ -2,6 +2,29 @@
 
 This page lists changes to the {product-name}.
 
+== Version 5.1.5
+
+=== New and Updated Features
+
+[cols="1,2", options="header"]
+|===
+| Feature | Details
+
+a|
+label:functionality[]
+label:new[]
+
+Added `neo4j.payload-mode` configuration property for source connector.
+| Introduced the `neo4j.payload-mode` option to define the structure of change messages. Available values are `COMPACT` and `EXTENDED`. `COMPACT` provides simpler messages but may have schema compatibility issues with property type changes, while `EXTENDED` includes type information to avoid such issues. Default is `EXTENDED`.
```

Comment:
```suggestion
| Introduced the `neo4j.payload-mode` option to define the structure of change messages. Available values are `COMPACT` and `EXTENDED`. `COMPACT` provides simpler messages but faces schema compatibility issues with property type changes, while `EXTENDED` includes type information to avoid such issues. Default is `EXTENDED`.
```